packages/enzyme-test-suite/test/RSTTraversal-spec.jsx

```diff
@@ -155,13 +153,6 @@ describe('RSTTraversal', () => {
       expect(nodeHasProperty(<div foo={-Infinity} />, 'foo', '-Infinity')).to.equal(true);
       expect(nodeHasProperty(<div foo={0} />, 'foo', '-Infinity')).to.equal(false);
     });
-
-    it('should throw when un unquoted string is passed in', () => {
```

Comment:
It wouldn't really be possible, this requires making a change directly to [the grammar for attribute values](https://github.com/gajus/scalpel/blob/master/src/grammar.ne#L86-L89) and there's no way to do that without changing `grammar.ne`.